aptos-move/aptos-vm/src/verifier/transaction_arg_validation.rs

```diff
@@ -245,18 +347,17 @@ fn get_len(cursor: &mut Cursor<&[u8]>) -> Result<usize, VMStatus> {
     }
 }
 
-//
-// Argument validation functions
-//
-
-// Check if a string is valid. This code is copied from string.rs in the stdlib.
-// TODO: change the move VM code (string.rs) to expose a function that does validation.
-fn check_string(s: &[u8]) -> Result<(), VMStatus> {
-    match std::str::from_utf8(s) {
-        Ok(_) => Ok(()),
-        Err(_) => Err(VMStatus::Error(
-            StatusCode::FAILED_TO_DESERIALIZE_ARGUMENT,
-            None,
-        )),
+fn serialize_uleb128(mut x: usize, dest: &mut Vec<u8>) {
```

Comment:
There is an implementation of this in `move_binary_format::file_format_common`. Maybe we should just open that up for use in clients. (We don't necessarily have to do it in this PR)